csfieldguide/templates/chapters/index.html

```diff
@@ -13,15 +13,24 @@ <h1 id="chapters" class="section-heading anchor-link">Chapters</h1>
 
 {% block content %}
 
-  {% if all_chapters %}
-      {% for chapter in all_chapters %}
-        <a href="{% url 'chapters:chapter' chapter.slug %}">
-          <img src="{% if chapter.icon %}{% static chapter.icon %}{% else %}http://placehold.it/238x200{% endif %}">
-          <h3>{{ chapter.name }}</h3>
-        </a>
-      {% endfor %}
-  {% else %}
-    <p>No chapters are available.</p>
-  {% endif %}
+  <div class="row chapter-list">
+
+      {% if all_chapters %}
+          {% for chapter in all_chapters %}
+          <div class="l12">
+            <div class="row">
+              <a href="{% url 'chapters:chapter' chapter.slug %}">
+                <img class="col l3 chapter-icon" src="{% if chapter.icon %}{% static chapter.icon %}{% endif %}">
```

Comment:
The sizing of this icon isn't working correctly because the `l3` class is setting the width to 25% of the container, while the `chapter-icon` at 20% (which seems to be 20% of the row but due to negative margins it is actually bigger than the `l3` class). To solve this problem, the image should be wrapped in the column class like so:

```html
<div class="col l3">
  <img class="chapter-icon responsive-img" src="{% if chapter.icon %}{% static chapter.icon %}{% endif %}">
</div>
```

And the `width: 20% !important;` should be removed from the `chapter-icon` class. The new class `responsive-img` sets the image to always be 100% of the parent.

Reply:
thanks